packages/electrode-archetype-react-app/i18next/index.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+var i18next = require("i18next");
+var Backend = require("i18next-xhr-backend");
+var LanguageDetector = require("i18next-browser-languagedetector");
+var reactI18nextModule = require("react-i18next");
+
+i18next
+  .use(Backend)
+  .use(LanguageDetector)
+  .use(reactI18nextModule)
+  .init({
+    detection: {},
+    fallbackLng: "en",
+    languages: ["en", "zh", "fr", "es", "it"],
+    react: {
+      wait: true
+    },
+    resources: {
+      en: { translation: require(`${__dirname}/../../../src/client/locales/en.json`) },
```

Comment:
you can't use `__dirname` because that depends on where the module will be installed under `node_modules`